lib/wallet.js

```diff
@@ -123,6 +137,70 @@ class Wallet extends ethers.Wallet {
 
         return [approvalTxReceipt, depositTxReceipt];
     }
+
+
+    /**
+      * Returns the wallet instance address
+      */
+    getAddress() {
+        return ethers.Wallet.prototype.getAddress.bind(this)();
```

Comment:
`bind(this)()` is essentially the same as `call(this)`

Reply:
Thanks, wasn't aware of `call`